pkg/cluster/networkprofile.go

```diff
@@ -0,0 +1,28 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"github.com/Azure/ARO-RP/pkg/util/feature"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+// PopulateMTUSize ensures that every cluster object has the MTUSize field defined
+func (m *manager) PopulateMTUSize(ctx context.Context) error {
```

Comment:
```suggestion
func (m *manager) populateMTUSize(ctx context.Context) error {
```

Should be called as a step at `pkg/cluster/install.go` in the `Install` function just after the FIPs step.